src/common/boards/nrf_PCA10188_0.1.0_54LV10.json

```diff
@@ -0,0 +1,124 @@
+{
+    "board": {
+        "boardVersion": "PCA10188",
+        "boardRevision": "0.1.0",
+        "boardName": "nRF54LV10 DK (hw rev. 0.1.0)"
+    },
+    "pins": [
+        {
+            "type": "vcom",
+            "id": "vcom0",
+            "name": "VCOM0",
+            "enable": {
+                "pin": 42
+            },
+            "hwfc": {
+                "pin": 20
+            }
+        },
+        {
+            "type": "vcom",
+            "id": "vcom1",
+            "name": "VCOM1",
+            "enable": {
+                "pin": 22
+            },
+            "hwfc": {
+                "pin": 23
+            }
+        },
+        {
+            "type": "switch",
+            "id": "ledcontrol",
+            "title": "Power to LEDs",
+            "label": "Disable to have LEDs unpowered (and kept off).",
+            "tooltip": "Enabling this option provides LEDs from LED0 to LED3 with 3.3 V of power.",
+            "enable": {
+                "pin": 45
+            }
+        },
+        {
+            "type": "switch",
+            "id": "swd-control",
+            "title": "Software Debugger (SWD)",
+            "label": "Disable to use an external debugger on the target chip.",
+            "tooltip": "When enabled, the SWD is connected and the IMCU is acting as the debugger. When disabled, the SWD is disconnected and you can use an external debugger connected to the Debug In header.",
+            "enable": {
+                "pin": 6
+            }
+        },
+        {
+            "type": "switch",
+            "id": "ls-en",
+            "title": "Load Switch",
+            "label": "Voltage level shifter",
+            "tooltip": "Enables voltage level shifter between Board Controller and nRF54L MCU when active. When disabled, the voltage level shifter is disconnected.",
+            "enable": {
+                "pin": 18
+            }
+        },
+        {
+            "type": "switch",
+            "id": "swo-control",
+            "title": "SWO Control",
+            "label": "Controls SWO signal",
```

Comment:
```suggestion
            "label": "Connect or disconnect the SWO signal",
```

Reply:
will it be correct to state "connect ... signal" in this case? can these words go together?

Reply:
Approved from HW designer